src/components/cesium_map/api/spatial.js

```diff
@@ -178,12 +178,14 @@ export class PointStreamDatasource extends Cesium.CustomDataSource {
           const p0 = Cesium.Cartesian3.fromDegrees(doc.x, doc.y, 10);
           this.entities.add({
             position: p0,
+            description: `<h4>Identifier: ${doc.id}</h4><span><b>All text fields</b>: ${doc.searchText}</span>`,
```

Comment:
`cesium infobox` uses `iframe` to display description so we could HTML code directly in the description.